lib/controller/maalcie/BeheerMaaltijdenController.class.php

```diff
@@ -257,6 +262,18 @@ public function leegmaken() {
 		redirect(maalcieUrl . '/prullenbak');
 	}
 
+	public function beoordelingen() {
+		if ($this->getMethod() == 'POST') {
```

Comment:
Je kunt deze splitsen in twee functies: POST_beheer en GET_beheer. Dan is deze if niet nodig.